### ISSUE TITLE
ur shit broke, file an error report isn't very helpful for debugging

### ISSUE THREAD

**Issue by mnapolitano89** [churchlab/millstone#597]
Server Error. Please file an issue at https://github.com/churchlab/millstone/issues/new. Describe what you were trying to do, and include the following offending URL: http://52.21.187.5/projects/cf1c3a61/samples

was trying to do: upload motherhueg samples

**Comment by mnapolitano89** [comment]
in this case it was an out of space error (probably)

**Comment by glebkuznetsov** [comment]
Good point Michael. It will be hard for people to communicate errors that occur on the server side.

Maybe one way to solve this is to have a dump logs button that just dumps the contents of the /var/log/supervisor/millstone-stdout

**Comment by mnapolitano89** [comment]
it also might be worth sanity checking uploaded sample sizes vs free space, and tossing a preemptive error if those samples are too big.

**Comment by glebkuznetsov** [comment]
I agree.
